src/app/modules/categories/movies.routes.ts

```diff
@@ -0,0 +1,11 @@
+import { SpecialCategoriesComponent } from './special-categories/special-categories.component';
```

Comment:
it should be categories-routing.module.ts. Similar to app.module and app-routing.module

Reply:
Done